docs/blog/blog.md

```diff
@@ -216,7 +216,7 @@ Encoding nominal categorical variables in unsupervised learning problems while o
 2. Login with your google account
 3. In this Spawn screen, select s2i-categorical-encoding-notebook:latest, in the memory size select Large ( Memory: 8Gi, CPU 2)
 4. Once your server starts, go into the directory named categorical-encoding-yyyy-mm-dd-hh-mm/demo
```

Comment:
:) Thanks for the catch. We should also remove _/demo_ from line 218 then (it's the old structure).

Reply:
:+1: